flowapi/flowapi/api_spec.py

```diff
@@ -0,0 +1,248 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import yaml
+from quart import Blueprint, request, jsonify, render_template, current_app
+from zmq.asyncio import Socket
+from flowapi import __version__
+
+blueprint = Blueprint("spec", __name__)
+
+
+def remove_discriminators(spec: dict) -> dict:
+    """
+    Remove any discriminator keys from an openapi spec dict.
```

Comment:
Can we add a sentence to the docstring about the purpose for removing discriminators?